docs/index-server.md

```diff
@@ -119,6 +119,16 @@ be written to file.
 The user can set the location for these file by using 'carbon.indexserver.temp.path'. By default
 table path would be used to write the files.
 
+## Prepriming
+The caching of the datamaps in the Index Server will start once the query is fired on the table for
+the first time. All the datamaps will be loaded if a count(*) query is fired and only the required
+will be loaded for any filter query. Unless the query is fired on the table there will be no caching, 
+which reduces the performance of the first time query. To improve the performance of the first time 
+query, cache can be preprimed in the Index Server. During prepriming the datamaps can be loaded in the 
+Index Server during load command before a select query is fired on the table.
+
+The user can enable prepriming by using 'carbon.indexserver.enable.prepriming'.
```

Comment:
Mention about the async call and that the load performance will not be degraded.

Reply:
done